### PR TITLE
Refact/main page

### PR DESCRIPTION
**Pull request by log1NVolcom**
# Pull Request Template
 
## Description
 
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
 
Fixes # (issue)
 
## Type of change
 
Please delete options that are not relevant.
 
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
 
## How Has This Been Tested?
 
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
 
- [ ] Test A
- [ ] Test B
 
**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:
 
## Checklist:
 
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/Main/Main.js [hunk 1]
```diff
@@ -95,311 +88,35 @@ class Main extends Component {
...
-            'Earth',
-            'Fire',
-            'Ice',
-            'Lightning',
-            'Restoration',
-            'Water',
-            'Wind'
-        ];
 
+        const tablePropsVars = {
```
[the remark this answers is not in the document]
Comment (log1NVolcom) [reply src/components/Main/Main.js]: my initial thoughts was to be one for vars and one for functions like tablePropsVars and tablePropsFunctions, but in this case is not worth

File: src/components/Main/Table/table.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+        sort,
+        orderAsc,
+        yokais,
+        tribe,
+        rank,
+        element,
+        name
+    } = tablePropsVars;
+    return (
+        <TableStyle>
```
[the remark this answers is not in the document]
Comment (log1NVolcom) [reply src/components/Main/Table/table.js]: Yes ofc, i will change the name

**Comment by joaopedrodcf** [comment]
Should we create an `index.js` file per component, for what I saw you didn't created it for the new components `Filter` `Search` and `Table`?

File: src/components/shared/table/Table.js [hunk 3]
```diff
@@ -5,36 +5,40 @@ import utils from '../../utils';
...
-                                    imageUrl={utils.getImage(
-                                        row.types,
-                                        row.wantedType
-                                    )}
-                                    altText={row.yokaiType}
-                                    size={row.size}
-                                />
-                            )}
-                        </td>
+        <table>
```
[the remark this answers is not in the document]
Comment (log1NVolcom) [reply src/components/shared/table/Table.js]: Lol, was a distraction! sorry

**Comment by joaopedrodcf** [comment]
As this is so behind in the history lets close it for now

**Pull request closed without merging**